packages/react-environment/src/webb-context/bridge/bridge-deposit.ts

```diff
@@ -42,7 +42,7 @@ export abstract class BridgeDeposit<T, K extends DepositPayload = DepositPayload
 
   abstract generateBridgeNote(
     mixerId: number | string,
-    destination: ChainId,
+    destination: number,
```

Comment:
I believe the typed destination (`ChainId`)is better than just a `number`

Reply:
well leave this for now